contract_variable_quantity/tests/test_contract_variable_quantity.py

```diff
@@ -4,10 +4,10 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import exceptions
-from odoo.tests.common import SavepointCase
+from odoo.tests.common import TransactionCase
 
 
-class TestContractVariableQuantity(SavepointCase):
+class TestContractVariableQuantity(TransactionCase):
     at_install = False
     post_install = True
```

Comment:
I think it should be used with the decorator `@odoo.tests.tagged("post_install", "-at_install")`